video_encoding/files.py

```diff
@@ -1,8 +1,10 @@
 import os
 
 from django.core.files import File
+from django.core.files.storage import default_storage
```

Comment:
It is possible to use a [specific storage](https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.FileField.storage) for a `FileField`. Hence I think we can't use the `default_storage` here and have to use the storage assigned to the field itself.